sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,995 @@
+to STX holders to propose a liveness recovery mechanism, whereby some BTC
+payouts are repurposed to fulfill withdraw requests should the custodian fail.
+
+* We reward Stackers for staying online to process 2-way peg operations by
+compensating them with BTC via PoX, regardless of the peg's operation volume.  Stackers
+are compensated proportional to their signing power on the peg (i.e how many
+STX they locked) regardless of how many deposit and withdrawal transactions they process.
+
+By leveraging these properties, sBTC offers the following advantages over the
+state-of-the-art:
+
+* **If a majority of Stackers are honest, then sBTC remains safe.**  Every BTC
+withdraw is paired with an equivalent, legitimate request from a wrapped BTC
+holder, _even if every miner is malicious_.  This is achieved by ensuring that
+all Stacking and wallet maintenance operations materialize in all Stacks forks.
+
+* **If a Byzantine fault-tolerant (BFT) majority of miners are honest, then sBTC
+remains live**.  All withdraw requests are eventually fulfilled, _even if every
+Stacker is malicious_.  This is because Stackers do not have a say in block
+production, and Stackers that do not fulfill withdraws lose their PoX-driven BTC
+income from miners.
+
+* **Withdraws of arbitrary amounts of BTC are fulfilled in
+a fixed amount of time on the happy path** if both Stackers and miners operate with a BFT honest
+majority.  If they do not, then withdraws of arbitrary amounts of BTC
+are fulfilled eventually by _redirecting_ Stackers' PoX payouts to fund withdraw
+requests.
+
+To support this, sBTC offers two modes of operation: **Normal mode** and **Recovery mode**.
+
+In **Normal mode**, the sBTC asset is
+backed 1:1 by BTC sent to a wallet controlled by a large fraction of Stackers,
+as measured by the fraction of locked STX they represent.  Each time BTC is
+sent to this wallet (a deposit operation), an equal number of sBTC are
+automatically minted to an address of the sender's choosing.  Stackers respond
+to withdraw requests by sending BTC to the requester's Bitcoin address, thereby
+maintaining the peg.  The act of responding to withdraw requests automatically
+destroys the requester's equivalent amount of sBTC on the Stacks chain.
+
+If the Normal mode encounters a liveness failure for any reason (including loss
+of BTC), the system transitions to a Recovery mode until enough Stackers come
+back online.
+
+In **Recovery mode**, a fraction of the PoX payouts are
+redirected to withdraw requests such that eventually, all outstanding requests
+will be fulfilled, even if the Stackers _never_ come back online.  While
+considerably slower than Normal mode, the design of Recovery mode ensures all
+sBTC can be redeemed so long as the Stacks blockchain and PoX are online.
+
+## Scope
+
+This proposal only describes Normal mode, and leaves Recovery mode to be described
+in a future SIP.
+
+This proposal, if ratified, would be executed by at least two hard forks:  one
```

Comment:
Add a specification `Maintenance` mode to encompass the need for manual recovery and is addressed later in the document with the details in https://github.com/stacks-network/sbtc/discussions/346 as mitigation strategies.

`Maintenance` mode is the temporary placeholder for `Recovery` mode prior to implementation.